src/core/utils.jl

```diff
@@ -163,7 +172,7 @@ function dictkeys_tosymbols(d::Dict)
             "monthly_totals_kwh",
             "prod_factor_series", 
             "monthly_energy_rates", "monthly_demand_rates",
-            "wholesale_rate", "blended_doe_reference_percents",
```

Comment:
why was "wholesale_rate" removed?

Reply:
I can't fully remember but looks like it was because wholesale_rate is also allowed to be a scalar Real, so in this case you get an error trying to convert to Array{Real, 1}. I actually don't really understand what the purpose of doing these conversions here is because isn't that done by specifying constructor argument types?